indico/modules/events/tracks/templates/_track_list.html

```diff
@@ -0,0 +1,46 @@
+{% from 'message_box.html' import message_box %}
+
+{% macro render_track_list(event) %}
+    {% if not event.tracks %}
+        {% call message_box('info', classes='no-rules-placeholder') %}
+            {% trans %}No tracks have been added yet.{% endtrans%}
+        {% endcall %}
+    {% else %}
+        <ul id="track-list" class="i-box-group vert" data-url="{{ url_for('.sort_tracks', event) }}">
```

Comment:
This need to be outside the macro. Currently this gets duplicated when we update via AJAX (since we set the `#track-list` content of the html from this macro.)